src/bundle/Resources/public/js/scripts/core/custom.tooltip.js

```diff
@@ -0,0 +1,28 @@
+(function(global, doc, eZ, $) {
+    eZ.helpers.tooltips = {
+        parse: () => {
+            const tooltipsNode = doc.querySelectorAll('[title]');
+            let delay, extraClasses;
+
+            for (tooltipNode of tooltipsNode) {
+                if (tooltipNode.title.length) {
+                    delay = {
+                        show: tooltipNode.getAttribute('data-delay-show') || 150,
+                        hide: tooltipNode.getAttribute('data-delay-hide') || 75,
+                    };
+                    extraClasses = tooltipNode.getAttribute('data-extra-classes') || '';
+
+                    $(tooltipNode).tooltip({
+                        delay,
+                        template: `<div class="tooltip ez-tooltip ${extraClasses}">
+                                            <div class="arrow ez-tooltip__arrow"></div>
+                                            <div class="tooltip-inner ez-tooltip__inner"></div>
+                                        </div>`,
+                    });
+                }
+            }
+        },
+    };
+
+    eZ.helpers.tooltips.parse();
```

Comment:
Not sure if helper itself should invoke the method.
The helper should be included with other helpers but invoked in the other file so we could move it as needed.